src/views/Project/issues.graphql

```diff
@@ -0,0 +1,24 @@
+fragment Issues on SearchResultItemConnection {
+    nodes{
```

Comment:
Use 2-space indentation.

Reply:
since this is related to the github one, i change it in other PR